babybear/src/main/java/frc/robot/commands/EveryBotPickerDefaultCommand.java

```diff
@@ -0,0 +1,62 @@
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// FRC - Charged Up - 2023
+// File: DefaultEveryBotPickerCommand.java
+// Intent: Forms a command to uptake/expell the cargo from the every bot picker.
+// ************************************************************
+
+// ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ 
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.EveryBotPickerSubsystem;
+
+import java.util.function.DoubleSupplier;
+
+public class EveryBotPickerDefaultCommand extends CommandBase {
```

Comment:
Similar comments as previously posted about picker.  Do we actually know what we need here?